tsconfig.json

```
@@ -26,6 +26,7 @@
             "@components": ["./src/components/index"],
             "@pages": ["./src/pages/index"],
             "@services": ["./src/services/index"],
+            "@providers": ["./src/providers/index"],
             "@utils": ["./src/utils/index"],
```

```suggestion
            "@utils": ["./src/utils/index"]
```
json does not have ending "," for last entry

---

json "does not" need to have ending "," for last entry?

---

if so, i gonna change for every other json in the folder as well